cartesi/explorer
# feat: Consume performance calc data from Subgraph

## Summary
Code changes to start consuming the performance of the staking pool from our Subgraph instead of the Postgraphile API that we will be retiring very soon. 

## Testing Instructions

1. Access the stake page.
2. Check the performance columns for the Staking pools are displaying.
3. (optional) Could check the dev tools network whether the calls are coming from:
  3.1. Mainnet: `https://thegraph.cartesi.io/subgraphs/name/cartesi/pos`
  3.2. Goerli: `https://thegraph.cartesi.io/subgraphs/name/cartesi/pos-goerli`
  3.3. It should not come from: `https://thegraph.cartesi.io/extended/*`

## Related Issue(s)

Closes #1
